public class Snippts11 {
    public static void main(String[] args) { 
        int[] arr = {1, 2, 3}; 
        // System.out.println(arr[5]);  //Exception in thread "main" java.lang.ArrayIndexOutOfBoundsException: Index 5 out of bounds for length 3
        System.out.println(arr[2]);
    } 
}




/*
 * 
 * Snippet 11:  
public class Main { 
    public static void main(String[] args) { 
        int[] arr = {1, 2, 3}; 
        System.out.println(arr[5]); 
    } 
} 
ï‚· What runtime exception do you encounter? Why does it occur?
 */